docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
 * You have created a Kafka instance with at least one Kafka topic in {product-kafka}.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
-. In the *Topics* page, click the name of the topic that you want to review or edit.
-+
-Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
-. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*.
+. Click the Kafka instance *Name* that contains the topic you want to configure.
```

Comment:
I think we can drop **Name** here. So, just "Click the Kafka instance that contains..."